CMakeLists.txt

```diff
@@ -745,10 +745,17 @@ endif()
              COMMAND sh -c "dd bs=2560 count=120 if=/dev/zero | $<TARGET_FILE:freedv_tx> 700D - - --testframes | $<TARGET_FILE:ch> - - --No -20 -f -10 | $<TARGET_FILE:freedv_rx> 700D - /dev/null --testframes --discard"
              )
 
+    # exercises complex rx codepath, albiet with just real samples
     add_test(NAME test_freedv_api_700D_AWGN_BER_USECOMPLEX
              COMMAND sh -c "dd bs=2560 count=120 if=/dev/zero | $<TARGET_FILE:freedv_tx> 700D - - --testframes | $<TARGET_FILE:ch> - - --No -20 -f -10 | $<TARGET_FILE:freedv_rx> 700D - /dev/null --testframes --discard --usecomplex"
              )
 
+    add_test(NAME test_freedv_api_700D_real_comp
```

Comment:
Do tests need to be added for 700E, etc. too?

Reply:
I think all OFDM modes share the two code paths (real and complex) that these tests are designed to exercise.

Reply:
Yep, we're good.